src/validation_workflow/validation_rpm.py

```diff
@@ -0,0 +1,26 @@
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import logging
+
+from validation_workflow.download_utils import DownloadUtils
+from validation_workflow.validation import Validation
+
+
+class ValidationRpm(Validation, DownloadUtils):
+
+    @classmethod
+    def download_artifacts(self, projects: list, version: str) -> bool:
+        for project in projects:
+            for architecture in ["x64", "arm64"]:
```

Comment:
Why is there still hardcoded architecture here?
Cant they be saved in the Validation<> class?